docs/quickstart.html

```diff
@@ -304,15 +304,24 @@
 </p>
 
 <pre>
-&gt; <b>cat /tmp/file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
+&gt; <b>bin/kafka-topics.sh --create \</b>
```

Comment:
Almost (up to wrong script name), but 287 is commented out.. and I think this order is slightly more natural.